programs/mango-v4/src/state/mango_account.rs

```diff
@@ -311,6 +311,7 @@ impl MangoAccount {
             perp_oo_count,
             token_conditional_swap_count,
         ) + (BORSH_VEC_SIZE_BYTES + size_of::<OpenbookV2Orders>() * usize::from(openbook_v2_count))
+            + BORSH_VEC_PADDING_BYTES
```

Comment:
I don't think this is right, the reserved bytes are not a vec and need no padding